modules/dreamview/backend/point_cloud/point_cloud_updater.cc

```diff
@@ -84,12 +86,6 @@ void PointCloudUpdater::RegisterMessageHandlers() {
       "RequestPointCloud",
       [this](const Json &json, WebSocketHandler::Connection *conn) {
         std::string to_send;
-        // If there is no point_cloud data for more than 2 seconds, reset.
```

Comment:
why is this removed?

Reply:
oh, i guess  we dont need to send empty point cloud if there is no update. i will use a flag to check if the current point cloud has been sent before and if so, skip send anything.  what do u think ?

Reply:
also i remove the localization estimate here since the timestamp is synced from simulation world

Reply:
this is for cleaning up point cloud if the user plays different rosbags with/without point clouds

Reply:
sure i will restore the old logic.